Dynatrace/dynatrace-operator
# Adds K8S object collection to support archive.

# Description

Add collection of selected K8S objects from the operator namespace and namespaces where injection has happened to the support archive.

## How can this be tested?
Deploy the operator and run:
`kubectl -n dynatrace exec pod/$(kubectl -n dynatrace get pods | grep dynatrace-operator | cut -f1 -d' ') -- /usr/local/bin/dynatrace-operator support-archive`

The command to retrieve the support archive will be printed on screen. Get it, unpack it and check the content.

## Checklist
